manifests/init.pp

```diff
@@ -201,6 +242,61 @@
     content => simp_version()
   }
 
+  file { '/usr/local/sbin/simp':
+    ensure => 'directory',
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0640'
+  }
+
+  if !$core_dumps {
+    pam::limits::add { 'prevent_core':
+      domain => '*',
+      type   => 'hard',
+      item   => 'core',
+      value  => '0',
+      order  => '100'
+    }
+  }
+
+  pam::limits::add { 'max_logins':
```

Comment:
Don't do this if `$max_logins` is `Undef`. Gives the user a method for getting rid of it. Alternatively, just add another selector.